### DISCUSSION TITLE
Differentiation between RCT1 and RCT2 paths

### DISCUSSION THREAD

**Discussion by Derpercoaster** [OpenRCT2/OpenRCT2#19049] [Feature requests]
Currently, it is impossible to tell RCT1 and RCT2 paths apart, since the sprites are identical in the path building window. It would be useful if there was some sort of indicator to differentiate them, such as this:
![Sandbox 2023-01-04 20-22-07](https://user-images.githubusercontent.com/60452987/210687615-54d06b6e-5aae-424f-bda1-43ac54aa4ef0.jpg)
The white boxes represent the indicators. They could just have a '1' or '2' in them (depending on the game the path is from), or a tiny icon (ie stairs or a slope).

**Comment by 733737** [comment]
this is the opposite of what I understand the devs want (a consolidation of pathing)

**Comment by ZeeMaji** [comment]
I like the latter idea of a tiny icon in the preview image, i think it'd streamline having both game's paths available without having to directly have the game's number in the thumbnail.

For all of the tarmacs & crazy paving it'd be stairs or slopes, whilst for the two dirty footpaths it would be a square corner vs a rounded corner.

**Comment by Lastorder-DC** [comment]
![demo](https://user-images.githubusercontent.com/18280396/210717546-b5872028-70ba-4dfe-8e23-bdf269406529.png)

Quick edit with RCT1 logo and OpenRCT2 logo

**Reply by karst** [reply]
I am curious if this is possible as currently 1px translates to 1px by default. Does look nice though

**Comment by ZeeMaji** [comment]
This has been somewhat implemented with the introduction of tooltips to the path selection dropdowns in #24070 as the objects clarify whether they have incline slopes or stairs, or for the dirty paths squared or rounded corners in their name and that's now displayed as tooltips.
